extensions/ql-vscode/src/view/result-tables.tsx

```diff
@@ -75,23 +77,66 @@ export class ResultTables
     return resultSets;
   }
 
+  private getResultSetNames(resultSets: ResultSet[]): string[] {
+    if (this.props.parsedResultSets.t === 'ExtensionParsed') {
+      return this.props.parsedResultSets.resultSetNames.concat([ALERTS_TABLE_NAME]);
+    }
+    else {
```

Comment:
And this branch is deprecated, right? To be removed when we only send paginated results? If so, can you mark that?